circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
See also https://mobile.twitter.com/abizern/status/852120220328218625 (= we are not alone and it's a general issue, as latest Xcode seems to be excluded from spotlight by default) and https://github.com/KrauseFx/xcode-install/issues/204 (discussion about possible alternatives)